Core/src/ServiceBuilder.php

```diff
@@ -30,7 +30,7 @@
 use Google\Cloud\Speech\SpeechClient;
 use Google\Cloud\Storage\StorageClient;
 use Google\Cloud\Trace\TraceClient;
-use Google\Cloud\Translate\TranslateClient;
+use Google\Cloud\Translate\V2\TranslateClient;
```

Comment:
Is it possible to leave this as `use Google\Cloud\Translate\TranslateClient`? It is a bit of a gray area, since the classes are identical in practice - but the returned type will vary. If anyone is doing typechecks they might fail.

Reply:
yes, but they will get the suppressed `E_USER_DEPRECATED` notice. I guess you're probably right though, this should be considered breaking.